README.md

```diff
@@ -83,13 +83,13 @@ Markdown can be expected to be lossy.
 [ZimWiki markup]: http://zim-wiki.org/manual/Help/Wiki_Syntax.html
 [TWiki markup]: http://twiki.org/cgi-bin/view/TWiki/TextFormattingRules
 [Haddock markup]: https://www.haskell.org/haddock/doc/html/ch03s08.html
-[groff man]: http://developer.apple.com/DOCUMENTATION/Darwin/Reference/ManPages/man7/groff_man.7.html
+[groff man]: https://www.gnu.org/software/groff/
 [Haskell]: https://www.haskell.org
 [GNU Texinfo]: http://www.gnu.org/software/texinfo/
 [Emacs Org mode]: http://orgmode.org
 [AsciiDoc]: http://www.methods.co.nz/asciidoc/
 [DZSlides]: http://paulrouget.com/dzslides/
-[Word docx]: http://www.microsoft.com/interop/openup/openxml/default.aspx
+[Word docx]: https://msdn.microsoft.com/en-us/library/gg278315.aspx
```

Comment:
The openxml link is difficult to find. Here's some info I have:

- Using wayback machine, the last known copy of the original link is in [Microsoft || Open Up](https://web.archive.org/web/20120306024812/http://www.microsoft.com/interop/openup/openxml/default.aspx) from 2012. This can be used to copy keywords to search for possible match (but can't seem to find a match from microsoft.com, so they probably have eliminated the page entirely).

- From within microsoft.com, the best I can find is [Office Open XML, ECMA-376, and ISO/IEC 29500](https://msdn.microsoft.com/en-us/library/office/gg607163(v=office.14%29.aspx).

- From iso.org: [ISO/IEC 29500-1:2016 - Information technology -- Document description and processing languages -- Office Open XML File Formats -- Part 1: Fundamentals and Markup Language Reference](http://www.iso.org/iso/home/store/catalogue_ics/catalogue_detail_ics.htm?csnumber=71691). This one requires a couple clicks and "I agree" to download the spec.

- From coverpages.org: [Cover Pages: Office Open XML File Formats Published as ISO/IEC 29500:2008 Final Standard.](http://xml.coverpages.org/ni2008-11-21-a.html). With some clicks, this one link back to the iso.org with the "I agree" button.